package.json

```diff
@@ -39,7 +39,8 @@
     "gulp": "^4.0.2",
     "gulp-stylelint": "^13.0.0",
     "mocha": "^8.0.1",
-    "node-sass": "^4.14.1",
+    "node-sass": "^5.0.0",
```

Comment:
The previous version of node-sass had some outdated dependencies (node-gyp), and I thought it best to build using current node-sass. I can revert this if necessary.